File: Python Word Examples/DocumentPropertiesApiDemo/getBuiltinDocumentProperties.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.document_properties_api.DocumentPropertiesApi(configuration)

file_name = 'GetDocumentBuiltinProperties.docx'
folder = 'input'
builtin_properties = api.get_builtin_document_properties(file_name, folder=folder)

File: JavaScript Word Examples/DocumentPropertiesApiDemo/getBuiltinDocumentProperties.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.DocumentPropertiesApi(apiClient);

    var filename = "GetBuiltinDocumentProperties.docx";
    var opt = { 'folder': "input" };
    var response = instance.getBuiltinDocumentProperties(filename, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();